src/Lykke.Service.ClientAssetRule/AutofacModule.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Autofac;
+using Common;
+using Common.Log;
+using Lykke.Service.Assets.Client;
+using Lykke.Service.ClientAssetRule.Rabbit.Subscribers;
+using Lykke.Service.ClientAssetRule.Settings;
+
+namespace Lykke.Service.ClientAssetRule
+{
+    public class AutofacModule : Module
+    {
+        private readonly AppSettings _settings;
+        private readonly ILog _log;
+
+        public AutofacModule(AppSettings settings, ILog log)
+        {
+            _settings = settings;
+            _log = log;
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder.RegisterInstance(_log)
+                .As<ILog>()
+                .SingleInstance();
+
+            builder.RegisterInstance<IAssetsService>(new AssetsService(new Uri(_settings.AssetsServiceClient.ServiceUrl)));
```

Comment:
AssetService has its own extension method for registration?

Reply:
No need all services registration in the ServiceCollectionExtensions